frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupItem/groupItem.component.tsx

```diff
@@ -92,7 +100,7 @@ export const GroupItem = ({ override, index }: GroupProps) => {
 	if (isString(group)) return (<CircularProgress />);
```

Comment:
There's a rerendering problem with the group items. When a new one gets created it is added to the list. Then all the list items refresh. This refresh seems unnecessary.

The image below shows 'group' being logged for a new group. As you can see it successfully reads as an object before it is updated to be a string before again returning as an object
![image](https://github.com/3drepo/3drepo.io/assets/62481250/57ff242f-e7a3-4b0d-b6a3-ef44787ee863)